vc_infer_pipeline.py

```diff
@@ -50,6 +51,24 @@ def change_rms(data1, sr1, data2, sr2, rate):  # 1是输入音频，2是输出
     return data2
 
 
+class F0Type(str, Enum):
+    Hz = "f0_hz"
+    Log = "f0_log"
+    Mel = "f0_mel"
+
+
+def hz2mel(f0):

```

Comment:
you can use directly the implementation in librosa.. (not important)

Reply:
I prefer to keep the RVC code unchanged. The numbers are slightly different between the two.